src/Auxiliary/Aux_Check.cpp

```diff
@@ -11,45 +11,46 @@ void Aux_Check( )
 {
 
    if ( OPT__CK_REFINE )
-      for (int lv=0; lv<NLEVEL-1; lv++)   Aux_Check_Refinement( lv, "DIAGNOSIS" );
+      for (int lv=0; lv<NLEVEL-1; lv++)                  Aux_Check_Refinement( lv, "DIAGNOSIS" );
 
    if ( OPT__CK_PROPER_NESTING )
-      for (int lv=1; lv<NLEVEL; lv++)     Aux_Check_ProperNesting( lv, "DIAGNOSIS" );
+      for (int lv=1; lv<NLEVEL; lv++)                    Aux_Check_ProperNesting( lv, "DIAGNOSIS" );
 
-   if ( OPT__CK_CONSERVATION )            Aux_Check_Conservation( "DIAGNOSIS" );
+   printf("Rank: %d, ConservedRefLoaded: %d\n", MPI_Rank, ConservedRefLoaded);
+   if ( OPT__CK_CONSERVATION || ! ConservedRefLoaded )   Aux_Check_Conservation( "DIAGNOSIS" );
```

Comment:
Although not always, it seems that the syntax with two spaces before and after `||` in the if statement is more common in GAMER.